bin/rmt-cli

```diff
@@ -2,8 +2,7 @@
 
 require 'etc'
 
-rmt_path = File.expand_path(File.join(File.dirname(File.realpath(__FILE__)), '..'))
-
+rmt_path = File.expand_path('..', File.join(__dir__))
```

Comment:
Nice improvement here

Reply:
Your comment made me realize that the `File.join` there is completely unnecessary haha. Removed.